command-line/lib/Command/Key/WalletId.hs

```diff
@@ -37,13 +37,17 @@ data Cmd = WalletId
 mod :: (Cmd -> parent) -> Mod CommandFields parent
 mod liftCmd = command "walletid" $
     info (helper <*> fmap liftCmd parser) $ mempty
-        <> progDesc "Get the wallet id from a given key"
+        <> progDesc "Shows the cardano-wallet WalletId for a given key"
```

Comment:
```suggestion
        <> progDesc "Shows the cardano-wallet wallet ID for a given key"
```

Reply:
I was trying to suggest that WalletId is not just any ID, but a specific type of ID.
Perhaps not then... suggestions applied.